packages/suite/src/components/suite/modals/index.tsx

```diff
@@ -262,24 +265,26 @@ const getModal = (props: SharedProps) => {
 
 // Action modal container component
 export const Modals = () => {
-    const props = useSharedProps();
+    const props: SharedProps = useSharedProps();
+
     const modalComponent = getModal(props);
+
     if (!modalComponent) {
```

Comment:
This `null` check isn't relevant anymore, am I right?

Reply:
done